apollo-router/src/services/external.rs

```diff
@@ -299,14 +301,43 @@ where
             .header(CONTENT_TYPE, "application/json")
             .body(router::body::from_bytes(serde_json::to_vec(&self)?))?;
 
+        let schema_uri = request.uri();
+        let host = schema_uri.host().unwrap_or_default();
+        let port = schema_uri.port_u16().unwrap_or_else(|| {
+            let scheme = schema_uri.scheme_str();
+            if scheme == Some("https") {
+                443
+            } else if scheme == Some("http") {
+                80
+            } else {
+                0
+            }
+        });
+        let path = schema_uri.path();
+
+        let http_req_span = tracing::info_span!(HTTP_REQUEST_SPAN_NAME,
```

Comment:
Let's use the Otel semantic conventions here: https://opentelemetry.io/docs/specs/semconv/http/http-spans/#http-client

Reply:
Updated to include required spans.

Reply:
Zipkin view with semconv in place: 
<img width="1047" alt="image" src="https://github.com/user-attachments/assets/c2ca146f-88b5-4339-83ea-4452c9306b23" />